src/NUnitEngine/nunit.engine/RuntimeFramework.cs

```diff
@@ -686,5 +687,82 @@ private static void AddMonoFramework(Version frameworkVersion, string profile)
         }
 
         #endregion
+
+        #region Helper Methods - .NET
+
```

Comment:
These methods moved to DotNetFrameworkLocator - is putting them back maybe a merge error?